docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/advanced-operations/snapshot.md

```diff
@@ -69,14 +66,18 @@ kubectl get pvc
 **Example Output**
 ```
 NAME                STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS     AGE
-ms-volume-claim     Bound    pvc-fe1a5a16-ef70-4775-9eac-2f9c67b3cd5b   1Gi        RWO            mayastor-1       15s
+ms-volume-claim     Bound    pvc-fe1a5a16-ef70-4775-9eac-2f9c67b3cd5b   1Gi        RWO            mayastor-3       15s
 ```
 
 :::note
 Copy the PVC name, for example, `ms-volume-claim`.
 :::
 
-4. (Optional) Create an application by following [these](../rs-deployment.md) steps.
+4. Create an application by following [these](../rs-deployment.md) steps.
+
+:::tip
+This step is optional.
+:::
```

Comment:
Let's not mention that this is optional. Volume will be consumed by application at some point that's what they are meant for.